moveit_ros/planning/planning_pipeline/test/planning_pipeline_tests.cpp

```diff
@@ -115,6 +117,34 @@ TEST_F(TestPlanningPipeline, NoPlannerPluginConfigured)
                std::runtime_error);
 }
 
+TEST_F(TestPlanningPipeline, TestTimeout)
+{
+  // construct pipline
+  EXPECT_NO_THROW(pipeline_ptr_ = std::make_shared<planning_pipeline::PlanningPipeline>(
+                      robot_model_, node_, "", PLANNER_PLUGINS, REQUEST_ADAPTERS, RESPONSE_ADAPTERS));
+
+  // WHEN generatePlan is called
+  // THEN A successful response is created
+  planning_interface::MotionPlanResponse motion_plan_response;
+  planning_interface::MotionPlanRequest motion_plan_request;
+  const auto planning_scene_ptr = std::make_shared<planning_scene::PlanningScene>(robot_model_);
```

Comment:
I think the successful case is already covered by earlier tests, so you can remove it here